src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -1,385 +1,195 @@
-        {text.substr(idx + highlightText.length)}
-      </span>
-    );
-  },
-
-  getLinkNode(team, project) {
-    let {organization} = this.props;
-    let label = this.getProjectLabel(team, project);
-
-    // TODO(billy): Only show platform icons for internal users
-    let internalOnly =
-      organization &&
-      organization.features &&
-      organization.features.includes('internal-catchall');
 
-    return (
-      <IdBadge
-        project={project}
-        avatarSize={16}
-        hideAvatar={!internalOnly}
-        displayName={<Link {...this.getProjectUrlProps(project)}>{label}</Link>}
-      />
-    );
-  },
-
-  renderProjectList({organization: org, projects, filter, hasProjectWrite}) {
-    const hasFilter = !!filter;
-    const hasProjects = projects && projects.length;
-    // Will always need to show divider
-    const showDivider = !hasFilter && hasProjectWrite;
-
-    if (hasProjects) {
-      return projects;
-    } else {
-      // There can be a filter and have no found results or
-      // there can simply be no projects to list
-      //
-      // Give an actionable item when there are no projects
-      return [
-        <MenuItem key="empty-message" className="empty-projects-item" noAnchor>
-          <div className="empty-message">
-            {hasFilter && t('No projects found')}
-            {!hasFilter && t('You have no projects.')}
-          </div>
-        </MenuItem>,
-        showDivider ? <MenuItem key="divider" divider /> : null,
-        !hasFilter && hasProjectWrite ? (
-          <MenuItem key="create-project" className="empty-projects-item" noAnchor>
-            <a
-              className="btn btn-primary btn-block"
-              href={`${this.urlPrefix()}/projects/new/`}
-            >
-              {t('Create project')}
-            </a>
-          </MenuItem>
-        ) : null,
-      ];
+    getProjectLabel(project) {
+      return project.slug;
     }
-  },
-
-  render() {
-    let org = this.props.organization;
-    let access = new Set(org.access);
-    let hasSingleTeam = org.teams.length === 1;
-
-    let projectList = sortArray(this.state.projectList, ([team, project]) => {
-      return [!project.isBookmarked, project.name];
-    });
 
-    let children = projectList.map(([team, project], index) => {
-      return this.getProjectNode(
-        team,
-        project,
-        this.state.filter,
-        hasSingleTeam,
-        this.state.currentIndex === index
-      );
-    });
-    const hasFilter = !!this.state.filter;
-    const hasProjects = children && !!children.length;
-    const dropdownClassNames = classNames('project-dropdown', {
-      'is-empty': !hasProjects,
-    });
+    handleSelect = ({value: project}) => {
+      let {router} = this.props;
+      let {to, href} = this.getProjectUrlProps(project);
+      if (to) {
+        router.push(to);
+      } else {
+        window.location.assign(href);
```

Comment:
why would this ever happen?

Reply:
good catch, the answer is..... django™™™™™ views!~

This is a good reminder though for me to figure out again which routes these are before deploying.